README.md

```diff
@@ -0,0 +1,7 @@
+# React-log-in-screen
+
+A basic working example of a login screen using react components to make a single page app.
+
+# How to install / use
+
+Clone the repository and make sure it's in it's own folder, run the index.html file and the app will open on your default web browser.
```

Comment:
When you say "run the index.html file" - this probably means different things depending on whether you are using Windows, linux or OS/X. In Windows are you meaning that people should double click the file? Or do you mean go to command line and type "index.html"+Enter ?